src/obfl-specification.html

```diff
@@ -1071,6 +1118,14 @@ <h4 class="include" id="L11470">dynamic-sequence</h4>
           <dd>Specifies a translation policy.</dd>
           <dd>See the <a href="#L2462">attribute description</a> for more
             information.</dd>
+		<dt>break-before [optional]</dt>
+          <dd class="no-markup">Inserts a break before the sequence begins.</dd>
+          <dd>This property only applies to text body sequences.</dd> 
+          <dd>One of 'sheet' or 'page'.</dd>
+		<dt>break-after [optional]</dt>
+          <dd class="no-markup">Inserts a break after the sequence has ended.</dd>
+          <dd>This property only applies to text body sequences.</dd> 
```

Comment:
See comment above. Also this doesn't even make sense, dynamic-sequence can only appear within pre- or post-content.

Reply:
@bertfrees Why is there a `break-before` attibute on dynamic sequences?

I have added the `break-after` here because there is also a `break-before`.

Why does it not make sense? Suppose you have a post-content with `break-after="page"` and the post-content ends on a right page. Then the next volume starts at a left page, instead of the default behavior.

Reply:
> Why is there a break-before attibute on dynamic sequences?

You mean why we are adding it? Because we want to control the type of page break. Not only for `sequence` but also for `dynamic-sequence` (and `toc-sequence`). So yes, both `break-before` and `break-after` make sense.

> Why does it not make sense?

I meant that the sentence "This property only applies to text body sequences" does not make sense in the context of `dynamic-sequence`, because a `dynamic-sequence` can never appear in the body.